src/ai/backend/manager/models/vfolder.py

```diff
@@ -2776,3 +2776,10 @@ async def get_permission_ctx(
         # )
         # permission_ctx.apply_host_permission_ctx(host_permission_ctx)
     return permission_ctx
+
+
+def is_vfsubpath_duplicate(vfsubpath: PurePosixPath, vfmounts: Iterable[VFolderMount]) -> bool:
+    for mount in vfmounts:
+        if vfsubpath.is_relative_to(mount.vfsubpath):
```

Comment:
Folder ID should be considered, as multiple folders with the same subpath may be incorrectly filtered, resulting in false positives.